docs/installation.md

```diff
@@ -0,0 +1,95 @@
+> ⚠️ This project is very alpha stage. Expect bugs and incomplete documentation.
+
+The following instructions are intended for contributors who want to try Metals
+and provide feedback. We do not provide support for day-to-day usage of Metals.
+
+# Global setup
+
+These steps are required once per machine.
+
+## Step 1 - sbt plugin
+
+The server needs to access some metadata about the build configuration. This
+data are produced by an sbt plugin. This plugin is currently not published, so
+you will need to copy paste it on your machine.
+
+Here's the source of the plugin:
+https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
+
+Copy the source to either (depending on your sbt version):
+
+* (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
+* (sbt 1.0) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
```

Comment:
I minor thing: some sbt users get confused about 1.+ versions. It's not very obvious that things for sbt 1.1 go to `~/.sbt/1.0/` instead of `~/.sbt/1.1/`. So I suggest to write here

```
* (sbt 1.+) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
```

Or `1.x` or `1.*`.